app/views/patients/show.html.erb

```diff
@@ -87,7 +89,12 @@
         <tr>
           <td>
             <div class="btn-group btn-group-sm" role="group">
-              <%= react_component('patient/laboratory/Laboratory', { authenticity_token: form_authenticity_token, patient: @patient, lab: lab }) %>
+              <%= react_component('patient/laboratory/Laboratory', {
+                                    authenticity_token: form_authenticity_token,
+                                    patient: @patient,
+                                    lab: lab,
+                                    onlyPosLab: @patient[:symptom_onset].nil? && lab[:result] == 'positive' && lab[:specimen_collection].present? && @num_pos_labs == 1
```

Comment:
can we make this snake case to be consistent with all other props coming from .erb files

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-38caf8aa9518c92857c2602080a58e672992506202b8d2b00adc4470dc9b10a2R96